.github/workflows/pr-check.yml

```diff
@@ -14,25 +14,35 @@ jobs:
         matrix:
           os: [windows-latest, ubuntu-latest]
       steps:
-      - name: Checkout from PR branch  
+      - name: Checkout from PR branch
         uses: actions/checkout@v2
-        with: 
+        with:
           repository: ${{ github.event.pull_request.head.repo.full_name }}
           ref: ${{ github.event.pull_request.head.ref }}
 
       - name: Installing node_modules
-        run: |
-          npm install
-          
+        run: npm install
+
       - name: Build GitHub Action
         run: npm run build
 
-      - uses: azure/login@v1
+      - name: Azure Login
+        uses: azure/login@v1
         with:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
-          
-      - uses: ./
+
+      # Deploy a DACPAC with only a table to server
```

Comment:
does the database get deleted after the test is done?

Reply:
Not currently. I looked into dropping it but because this action runs in parallel on Linux and Windows, I ran into race conditions occasionally when either SqlPackage or SQLCMD hasn't finished and the DB is already dropped by the other run.

I'm not sure what's the best way to cleanup here. Maybe we can add a scheduled job to delete the DB at midnight or something.

Reply:
Is it possible for the database name to be different for Linux and Windows? Otherwise a scheduled job sounds good! It would be good to clean up the database so the subscription doesn't get charged for it when it isn't being used and it's a small dacpac being deployed anyway.

Reply:
Added test DB name as an env variable. Also added a cleanup step to drop the DB. The cleanup step has to be separate because it needs connection to master DB.